cac-prod.yml

```diff
@@ -52,4 +52,16 @@ jenkins:
           ^HMCTS_.*IAC.*\/(ia-case-api|ia-shared-infrastructure|ia-ccd-e2e-tests)\/master
         name: IA
         recurse: true
-        title: I & A Dashboard
\ No newline at end of file
+        title: I & A Dashboard
+    - buildMonitor:
+        includeRegex: >-
+          ^HMCTS_.*probate.*\/(probate-frontend|probate-persistence-service|probate-business-service|probate-submit-service|probate-back-office)\/(master|demo|cnp)
+        name: PROBATE
+        recurse: true
+        title: PROBATE Dashboard
+    - buildMonitor:
+        includeRegex: >-
+          ^HMCTS_.*intestacy.*\/(intestacy-frontend|intestacy-orchestrator-service)\/(master|demo|cnp)
```

Comment:
this doesn't seem to exist?

Reply:
they are starting. Not yet pushed the project yet